ffmpeg/__init__.py

```diff
@@ -220,9 +220,11 @@ def get_args(self):
         args += reduce(operator.add, [self._get_output_args(node, stream_name_map) for node in output_nodes])
         args += reduce(operator.add, [self._get_global_args(node) for node in global_nodes], [])
         return args
-
-    def run(self):
-        args = ['ffmpeg'] + self.get_args()
+    
+    def run(self, cmd='ffmpeg'):
+        if type(cmd) == str:
```

Comment:
About the `else` you may want to check it is a supported iterable, but generally I don't do too many checks, the interpreter gives understandable tracebacks.
E.g.:

```
TypeError: unsupported operand type(s) for +: 'int' and 'list'
```